### PR TITLE
Updating admin routing to comply with new Magento security patch

### LINKED ISSUE #142: SUPEE-6788 / APPSEC-1034 Admin Routing Updates
Magento's [SUPEE-6788 / APPSEC-1034](http://magento.com/security/patches/supee-6788-technical-details) security patch requires all extensions to use proper routes for admin controllers.  It appears that the Reverb controllers will need to be updated to close this security hole.

This is based on the current admin URLs being in the form of: `https://{base_url}/reverbSync/...` whereas they'll need to use the `admin` router: `https://{base_url}/admin/reverbSync/...`

### PR DESCRIPTION
**Pull request by dunagan5887**
Fix #142

### CODE REVIEW

File: app/code/community/Reverb/ReverbSync/Helper/Orders/Retrieval/Update.php [hunk 1]
```diff
@@ -8,7 +8,16 @@ class Reverb_ReverbSync_Helper_Orders_Retrieval_Update extends Reverb_ReverbSync
...
-    const MINUTES_IN_PAST_FOR_UPDATE_QUERY = 11;
+
+
+
+
+    const MINUTES_IN_PAST_FOR_UPDATE_QUERY = 110000;
+
+
+
+
```
Comment (skwp) [inline app/code/community/Reverb/ReverbSync/Helper/Orders/Retrieval/Update.php]: this number got crazy large and there's a huge number of spaces here. i'm guessing not intentional?

File: app/code/community/Reverb/ReverbSync/Model/Sync/Order/Update.php [hunk 2]
```diff
@@ -41,6 +41,19 @@ protected function _executeStatusUpdate($magento_order_entity_id, $reverb_order_
     {
         try
         {
+
+
+
+
+            $reverb_order_status = 'paid';
+
+
```
Comment (skwp) [inline app/code/community/Reverb/ReverbSync/Model/Sync/Order/Update.php]: ???

File: app/code/community/Reverb/ReverbSync/Helper/Orders/Retrieval/Update.php (hunk 1 above)
Comment (dunagan5887) [inline app/code/community/Reverb/ReverbSync/Helper/Orders/Retrieval/Update.php]: That's embarassing, definitely wasn't supposed to commit three of these files. Reject this PR

**Comment by skwp** [comment]
please re-open with clean files

**Pull request closed without merging**